kuvaselaamo/locale/en/LC_MESSAGES/django.po

```diff
@@ -627,6 +627,10 @@ msgstr "Feedback submitted successfully"
 msgid "Feedback error"
 msgstr "Error occurred while submitting. Try again after a while"
 
+msgid "Feedback temp"
+msgstr "Unfortunately, the processing of feedback is currently very congested due to the volume of feedback and "
+       "response times are longer than normal. We apologize for the situation!"
```

Comment:
Don't know if Aki provided these translations or not, but I would say that this expression doesn't sound idiomatic in English. "We apologise for the inconvenience" or something to that degree would sounds better to my ear. Perhaps check this with Jonatan ?

Reply:
Texts weren't provided. Both Aki and Jonathan approved these, all though your suggestion sounds good. I'll toss them another message.

Reply:
If they have already OK'd these, then they are fine for me as well :)

Reply:
Aki approved the change so I will be making it before merging.